pkg/conditions/composite/handler.go

```diff
@@ -0,0 +1,78 @@
+package composite
```

Comment:
This handler type is used to create handlers for `Cluster` and `MachinePool` (by combining handlers for required conditions) that are then added to their respective controllers.